core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -45,15 +45,16 @@
 #ifndef KOKKOS_SYCL_INSTANCE_HPP_
 #define KOKKOS_SYCL_INSTANCE_HPP_
 
-#include <Kokkos_SYCL.hpp>
+#include <memory>
+#include <CL/sycl.hpp>
 
 namespace Kokkos {
 namespace Experimental {
 namespace Impl {
 
 class SYCLInternal {
  public:
-  using size_type = Kokkos::Experimental::SYCL::size_type;
+  using size_type = unsigned int;
```

Comment:
Why did you change this?

Reply:
This allowed me to resolve some circular dependency problems.

Reply:
I can defer to the Instance class to only define it in one place.